deployment/stacks/deployFrontend.ts

```diff
@@ -0,0 +1,159 @@
+import { Construct } from "constructs";
+import {
+  aws_iam as iam,
+  aws_cloudfront as cloudfront,
+  Duration,
+  aws_s3 as s3,
+  aws_s3_deployment as s3_deployment,
+  custom_resources as cr,
+} from "aws-cdk-lib";
+import path from "path";
+import { execSync } from "node:child_process";
+
+interface DeployFrontendProps {
+  scope: Construct;
+  stage: string;
+  uiBucket: s3.Bucket;
+  distribution: cloudfront.Distribution;
+  apiGatewayRestApiUrl: string;
+  applicationEndpointUrl: string;
+  identityPoolId: string;
+  userPoolId: string;
+  userPoolClientId: string;
+  userPoolClientDomain: string;
+  iamPermissionsBoundary: iam.IManagedPolicy;
+  iamPath: string;
+  customResourceRole: iam.Role;
+  launchDarklyClient: string;
+  s3AttachmentsBucketName: string;
+  redirectSignout: string;
+}
+
+export function deployFrontend(props: DeployFrontendProps) {
+  const {
+    scope,
+    stage,
+    distribution,
+    apiGatewayRestApiUrl,
+    applicationEndpointUrl,
+    identityPoolId,
+    userPoolId,
+    userPoolClientId,
+    userPoolClientDomain,
+    iamPermissionsBoundary,
+    iamPath,
+    uiBucket,
+    launchDarklyClient,
+    s3AttachmentsBucketName,
+    redirectSignout,
+  } = props;
+
+  const reactAppPath = "./services/ui-src/";
+  const buildOutputPath = path.join(reactAppPath, "build");
+  const fullPath = path.resolve(reactAppPath);
+
+  execSync("SKIP_PREFLIGHT_CHECK=true yarn run build", {
+    cwd: fullPath,
+    stdio: "inherit",
+  });
```

Comment:
The skip_preflight_check is missing from seds and hcbs. Is this the only place that is used? If so let me know and I'll change them to match here and rip it out of .env.tpl

Reply:
I'm just matching MFP how it was in SLS.